src/main/java/bio/terra/service/dataset/DatasetJsonConversion.java

```diff
@@ -182,14 +183,14 @@ public static TableModel tableModelFromTable(DatasetTable datasetTable) {
     public static Column columnModelToDatasetColumn(ColumnModel columnModel) {
         return new Column()
                 .name(columnModel.getName())
-                .type(columnModel.getDatatype())
+                .type(columnModel.getDatatype().toString())
                 .arrayOf(columnModel.isArrayOf());
     }
 
     public static ColumnModel columnModelFromDatasetColumn(Column tableColumn) {
         return new ColumnModel()
                 .name(tableColumn.getName())
-                .datatype(tableColumn.getType())
+                .datatype(TableDatatypes.fromValue(tableColumn.getType()))
                 .arrayOf(tableColumn.isArrayOf());
     }
```

Comment:
Sorry, my earlier comment was a bit vague. This is what I have in mind
```
    public ColumnModel toColumnModel() {
        return new ColumnModel()
                .name(name)
                .datatype(TableDatatypes.fromValue(type))
                .arrayOf(isArrayOf);
    }
```
In general, when I see a static method on an object X which also accepts X as a parameter, it's usually the case that it would be better expressed as a non-static method on X that operates on `this` instead of a parameter.